backend/src/test/java/de/neuefische/backend/controller/RestCountriesControllerTest.java

```diff
@@ -0,0 +1,107 @@
+package de.neuefische.backend.controller;
+
+import de.neuefische.backend.RestCountriesWebClient;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class RestCountriesControllerTest {
+    String endPoint = "/api/countries";
+
+    @Autowired
+    private MockMvc mvc;
+
+
+    @Test
+    void get_countries_returns_a_list_of_countries_and_deserialize()
```

Comment:
the convention is <method-name>_when<initial-values>_then<expected-outcome>

```suggestion
    void getCountries_returnsAListOfCountriesAndDeserialize()
```